frame/collective/src/lib.rs

```diff
@@ -515,6 +515,9 @@ decl_module! {
 			let position_yes = voting.ayes.iter().position(|a| a == &who);
 			let position_no = voting.nays.iter().position(|a| a == &who);
 
+			// first vote of member in the motion
+			let is_account_voting_first_time = position_yes.is_none() && position_no.is_none();
```

Comment:
This behavior should be mentioned in the docs above

Reply:
Updated the function header Doc string ...